managed/services/supervisord/supervisord.go

```diff
@@ -680,6 +680,28 @@ stdout_logfile_backups = 3
 redirect_stderr = true
 {{end}}
 
+{{define "vmproxy"}}
+[program:vmproxy]
+priority = 7
```

Comment:
Too many programs have the same `priority = 7` value. Let's give it priority 9, so we don't end up guessing who should start when.